reVX/least_cost_xmission/aoswt_utilities.py

```diff
@@ -0,0 +1,604 @@
+"""
+Various utility functions to prep data for AOSWT processing.
```

Comment:
Would this module not work for the pacific? Why not?

Reply:
Looks like Mike was in the process of working on this (see quote form him below), but the demand for this work dropped off. I propose to shelve this question until demand for this work arises once more


> TODO:
> 
> * When saving paths, also export to CSV
> * calculate % path distance offshore vs land and save to dist_offshore_frac column
> * change README to rst
> * remove AOSWT from naming when appropriate and make ocean agnostic
> * Update linter config (done)
> * Make 500kV default voltage for fake t-lines

Reply:
The AOSWT code would really work for determining paths anywhere within the reV raster, including on land.

Reply:
Should we rename the file?

Reply:
The code was written for AOSWT (Atlantic off-shore wind transmission) and has only been used for AOSWT so far. Naming it after it provides some history, assuming the user knows what AOSWT is. Keeping it named AOSWT until it's used for something else makes sense to me. That said, I won't be offended if you decide to change it.